ddtrace/llmobs/_integrations/base.py

```diff
@@ -204,12 +204,13 @@ def llmobs_set_tags(
         kwargs: Dict[str, Any],
         response: Optional[Any] = None,
         operation: str = "",
+        history: List[Any] = [],
```

Comment:
I added another argument here to pass in the history of a chat.I also considered trying to prepend the history to the contents argument; however they have different types (`list[Content]` vs `ContentsType`) which would make the extraction logic more convoluted.